SwissKnife/gcloud/GCloudStorage.py

```diff
@@ -158,21 +159,26 @@ def get_storage_complete_file_path(file_name: str,
         :return: Complete path of a file stored in gcloud
         :rtype: str
         """
-        bucket_name = f'{BUCKET_NAME}/' if with_bucket else ''
-        file_prefix = f'{BUCKET_PATH_PREFIX}/' if with_prefix and BUCKET_PATH_PREFIX else ''
-        path = f'{file_path}/' if file_path else ''
+        bucket_name = BUCKET_NAME if with_bucket else ''
+        file_prefix = BUCKET_PATH_PREFIX if with_prefix and BUCKET_PATH_PREFIX else ''
+        final_file_path = file_path if file_path else ''
         gs_prefix = 'gs://' if with_gs else ''
+        final_file_name = file_name if file_name else ''
         
-        return f'{gs_prefix}{bucket_name}{file_prefix}{path}{file_name}'
+        return os.path.join(gs_prefix, bucket_name, file_prefix, final_file_path, final_file_name)
```

Comment:
I have a concern about this: `os.path.join` uses the system path separator (in Windows `\`). ¿Do we know if GCS supports such a symbol. Otherwise this code is not cross platform.

Reply:
wow, good question. In Linux works fine.

Reply:
I usually would not care because we run everything on containers. But this is open source so someone could run this on Windows.